extra-files/cloud_setup/usegalaxy_main_tool_list.yml

```diff
@@ -0,0 +1,2015 @@
+  install_resolver_dependencies: True
+- name: logistic_regression_vif
+  owner: devteam
+  revisions:
+  - bd196d7c1ca9
+  tool_panel_section_label: Multiple regression
+  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+  install_resolver_dependencies: True
+- name: partialr_square
+  owner: devteam
+  revisions:
+  - 88ef41de020d
+  tool_panel_section_label: Multiple regression
+  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+  install_resolver_dependencies: True
+- name: rcve
+  owner: devteam
+  revisions:
+  - 7740956d197b
+  tool_panel_section_label: Multiple regression
+  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+  install_resolver_dependencies: True
+- name: best_regression_subsets
+  owner: devteam
+  revisions:
+  - 4f33ec73e445
+  tool_panel_section_label: Multiple regression
+  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+  install_resolver_dependencies: True
+- name: canonical_correlation_analysis
+  owner: devteam
+  revisions:
+  - 9bc0c48a027f
+  tool_panel_section_label: Multivariate Analysis
+  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+  install_resolver_dependencies: True
+- name: kernel_canonical_correlation_analysis
+  owner: devteam
+  revisions:
+  - 7a092113eb8c
+  tool_panel_section_label: Multivariate Analysis
+  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+  install_resolver_dependencies: True
+- name: kernel_principal_component_analysis
+  owner: devteam
+  revisions:
+  - e9ebd4bfbdfc
+  tool_panel_section_label: Multivariate Analysis
+  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+  install_resolver_dependencies: True
+- name: principal_component_analysis
+  owner: devteam
+  revisions:
+  - f568051cdf2e
+  tool_panel_section_label: Multivariate Analysis
+  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+  install_resolver_dependencies: True
+- name: meme_fimo
+  owner: iuc
+  revisions:
+  - eca84de658b0
+  tool_panel_section_label: Motif Tools
+  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+  install_resolver_dependencies: True
+- name: meme_meme
+  owner: iuc
+  revisions:
+  - 62d1fae3b7d3
+  tool_panel_section_label: Motif Tools
+  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+  install_resolver_dependencies: True
+- name: str_fm
+  owner: arkarachai-fungtammasan
+  revisions:
+  - 48b5f719e36a
+  tool_panel_section_label: 'STR-FM: Microsatellite Analysis'
+  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+  install_resolver_dependencies: True
+- name: str_fm
+  owner: arkarachai-fungtammasan
+  revisions:
+  - 48b5f719e36a
+  tool_panel_section_label: 'STR-FM: Microsatellite Analysis'
+  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+  install_resolver_dependencies: True
+- name: table_recalibration
+  owner: devteam
+  revisions:
+  - 30e1dd77e99c
+  tool_panel_section_label: 'NGS: GATK Tools (beta)'
+  tool_shed_url: https://toolshed.g2.bx.psu.edu/
+  install_resolver_dependencies: True
+##########
+# The Following list of tools in the commented section DO NOT install cleanly.
```

Comment:
They typically stay in either cloning state or have missing dependencies - we have not looked into the details of each yet.